k8s/k8s.go

```diff
@@ -356,7 +361,12 @@ func CmdDelK8s(c *calicoclient.Client, ep api.WorkloadEndpointMetadata, args *sk
 
 	// Return the IPAM error if there was one. The IPAM error will be lost if there was also an error in cleaning up
 	// the device or endpoint, but crucially, the user will know the overall operation failed.
-	return ipamErr
+	if ipamErr != nil {
+		return ipamErr
+	}
+
+	logger.WithField("endpoint", wep).Info("Endpoint deletion complete.")
```

Comment:
I don't think this is the right log - Endpoint deletion was complete way up above, this log should say that all of tearDown is complete.